README.md

```diff
@@ -1,5 +1,7 @@
-# Jane
-
+<p align="center">
+    <img width="500" height="180" src="https://github.com/Korbeil/janephp/tree/master/identity/logo_jane_full.svg" alt="Jane logo" />
```

Comment:
You need to use the url from raw.githubusercontent because it's not replaced when not an image markdown ;)